src/main/java/com/softexpert/raffle/FileReader.java

```diff
@@ -0,0 +1,29 @@
+package com.softexpert.raffle;
+
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReader {
+
+	public List<String> readFile(String fileName) throws IOException {
+		InputStream inputStream = new FileInputStream(fileName);
```

Comment:
-1 usa o guava, código ficaria assim: "List<String> result = Files.readLines(file, Charsets.UTF_8);"  http://www.baeldung.com/guava-write-to-file-read-from-file